tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -44,6 +45,11 @@ Define {
 	DMIC0_PCM_CAPS			'Passthrough Capture 13'
 	DMIC0_PIPELINE_STREAM_NAME	'copier.DMIC.14.1'
 	PLATFORM 			"none"
+	DEEP_BUFFER			0
+	DEEP_BUFFER_PIPELINE_ID		15
+	DEEP_BUFFER_PCM_ID		3
```

Comment:
this is going to be problematic, you need to avoid conflicts. For SoundWire, the PCM_ID 3 is not a good idea, this can be the amplifier reference.

We've used device 31 in the past.

[PCMDeviceList.txt](https://github.com/thesofproject/sof/files/9940844/PCMDeviceList.txt)

Reply:
sure, changed to 31